src/main/kotlin/no/nav/familie/ef/sak/beregning/BeregningController.kt

```diff
@@ -74,7 +75,8 @@ class BeregningController(private val stegService: StegService,
     @GetMapping("/{behandlingId}")
     fun hentBeregnetBeløp(@PathVariable behandlingId: UUID): Ressurs<List<Beløpsperiode>> {
         tilgangService.validerTilgangTilBehandling(behandlingId, AuditLoggerEvent.UPDATE)
-        return Ressurs.success(tilkjentYtelseService.hentForBehandling(behandlingId).tilBeløpsperiode())
+        val startDatoForVedtak = vedtakService.hentVedtak(behandlingId).perioder?.perioder?.minByOrNull { it.datoFra }?.datoFra
+        return Ressurs.success(tilkjentYtelseService.hentForBehandling(behandlingId).tilBeløpsperiode(startDatoForVedtak))
```

Comment:
Burde denne kanskje få en test, både for en med dato, og ikke med?

Reply:
Enig!